lapis2/src/test/kotlin/org/genspectrum/lapis/controller/LapisControllerTest.kt

```diff
@@ -40,4 +41,92 @@ class LapisControllerTest(@Autowired val mockMvc: MockMvc) {
             .andExpect(status().isOk)
             .andExpect(jsonPath("\$.count").value(0))
     }
+
+    @Test
+    fun `GET nucleotideMutationProportions without explicit minProportion defaults to 0,05`() {
```

Comment:
```suggestion
    fun `GET nucleotideMutationProportions without explicit minProportion defaults to 0.05`() {
```

Reply:
Unfortunately that is not valid Kotlin syntax... That's why I went for the `,` as a workaround. Do you have a better suggestion?

Reply:
oh. Would 5% be valid?

Reply:
5e-2? 😅

Reply:
According to IntelliJ, `%` can cause problems on windows, so I went for `5 percent` :+1: